src/pages/profile-page/ui/profile-page.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useStore } from "effector-react";
+
+import { sessionModel } from "entities/session";
+
+import { Container, Typography } from "shared/ui";
+import { LogoutButton } from "features/session";
+
+export const ProfilePage: React.FC = () => {
+  const user = useStore(sessionModel.$user);
+  const loading = useStore(sessionModel.getUserFx.pending);
```

Comment:
technically ok but better would be preparing own state

Reply:
loading removed for unnecessarity